monitoring-support/src/main/java/org/terracotta/monitoring/PlatformClientFetchedEntity.java

```diff
@@ -36,13 +35,13 @@
 
   public String clientIdentifier;
   public String entityIdentifier;
-  public transient ClientDescriptor clientDescriptor;
+  public transient Object clientDescriptor;
```

Comment:
Java6/8 madness.  I will go with a more invasive change.  Confirm please that you are using these objects only server-side and java8?